src/views/v2/commerce/success.php

```diff
@@ -29,7 +29,7 @@
 		<?php $this->template( 'order/header' ); ?>
 		<?php $this->template( 'order/description' ); ?>
 		<?php $this->template( 'order/details' ); ?>
-		<?php $this->template( 'order/attendees' ); ?>
+		<?php tribe( 'tickets.main' )->get_component_template()->template( 'your-tickets/attendees', $this->get_global_values(), true ); ?>
```

Comment:
If we're using the compmonents template, then maybe we should get rid of the `order/attendees` templates, as they won't be used.